DemoQA/Pages/registrationFormPage.js

```diff
@@ -0,0 +1,41 @@
+import { Selector, t } from "testcafe";
+import { myUserDetails } from "../Constants/userInformation";
+import mainPage from "../Pages/mainPage";
+import webTablesPage from "../Pages/webTablesPage";
+
+
+class RegistrationFormPage {
+    constructor () {
+        this.firstNameInput = Selector("#firstName");
+        this.lastNameInput = Selector("#lastName");
+        this.emailInput = Selector("#userEmail");
+        this.ageInput = Selector("#age");
+        this.salaryInput = Selector("#salary");
+        this.departmentInput = Selector("#department");
+        this.submitBttn = Selector("#submit");
+    }
+
+    async registrationFormInformation(user) {
```

Comment:
Start function names with a verb and name the function in a way that easily describes it's purpose.